packages/attest/src/store.ts

```diff
@@ -1,7 +1,12 @@
 import * as github from '@actions/github'
+import {retry} from '@octokit/plugin-retry'
 
 const CREATE_ATTESTATION_REQUEST = 'POST /repos/{owner}/{repo}/attestations'
+const DEFAULT_RETRY_COUNT = 3
```

Comment:
Mentioned above, but also for posterity: let's make this `5`

Reply:
👍